openplayground-api/src/openplayground.py

```diff
@@ -72,4 +72,61 @@ def generate(self, model, prompt, **kwargs):
 
       data = json.loads(matches[1])
       data["event"] = matches[0]
-      yield data
\ No newline at end of file
+      yield data
+
+class Auth:
+    # sign in
+    api_url = "https://clerk.nat.dev/v1/client/sign_ins/"
+
+    def __init__(self):
+        self.session = requests.Session()
+        self.headers = {
+            # "Host": "clerk.nat.dev",
```

Comment:
Don't comment out the host and user-agent headers, cause then the user-agent will end up defaulting to `python-requests/2.25.0` or something similar.